app/components/documents/document-details-panel.js

```diff
@@ -134,7 +134,12 @@ export default class DocumentsDocumentDetailsPanel extends Component {
   }
 
   verifyDeleteSignFlow = task(async () => {
-    await this.signatureService.removeSignFlow(this.args.piece);
+    const signMarkingActivity = await this.args.piece.belongsTo('signMarkingActivity').reload();
+    if (signMarkingActivity) {
```

Comment:
@sergiofenoll apparently you can use `removeSignFlowForPiece` in the service to do exactly this check...

Reply:
oops, apparently that is only for MARKED signflows.